core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -129,6 +129,7 @@ class SnappySession(@transient private val sc: SparkContext,
   private[spark] val snappyContextFunctions = sessionState.contextFunctions
 
   SnappyContext.initGlobalSnappyContext(sparkContext, this)
+  sessionState.functionRegistry.registerFunction("DSID", e => DSID())
```

Comment:
Add this to a method "object functions" in the separate file as suggested above.